src/test/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndexTest.kt

```diff
@@ -391,4 +393,46 @@ class BuildKmerIndexTest {
         assertEquals(truth,inputFile)
     }
 
+    @Test
+    fun testCreateAgcCommandLists() {
+        val graph = HaplotypeGraph(listOf(TestExtension.smallseqLineAHvcfFile,
```

Comment:
It might be nice to have a brief sentence saying what each graph contains or is missing, just for clarity when analyzing the asserts.

Reply:
I added comments